test/e2e-tests.sh

```diff
@@ -73,6 +73,13 @@ function knative_setup() {
     popd
   fi
   wait_until_pods_running knative-eventing || fail_test "Knative Eventing did not come up"
+
+  # TODO install head if !is_release_branch
```

Comment:
Just curious, do we have plan to move `monitoring` out of `serving`?

Reply:
I think so, but that may take sometime.